types.go

```diff
@@ -1402,11 +1483,24 @@ func euiToString(eui uint64, bits int) (hex string) {
 	return
 }
 
+// copyBytes returns a copy of a byte slice.
+func copyBytes(b []byte) []byte {
```

Comment:
this is a too thin of a wrapper of `copy` right now. Please just keep copyIP and copy the netmask below manually